st2tests/st2tests/fixtures/packs/dummy_pack_23/actions/workflows/__init__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 The StackStorm Authors.
```

Comment:
`__init__.py` should be empty unless there is logic in the file. Please delete all these headers.

Reply:
Deleted all the headers. Now __init__.py is empty